client/gutenberg/extensions/markdown/editor.scss

```diff
@@ -8,7 +8,7 @@ $light-gray-500: #e2e4e7;
 $text-editor-font-size: inherit;
 
 .wp-block-jetpack-markdown__placeholder {
-	opacity: 0.5;
+	opacity: 0.62;
```

Comment:
I wish we could use a SASS var or something here. Manually updating this to follow Gutenberg/core isn't exactly great maintanability :slightly_frowning_face: 

Maybe at least add an inline comment pointing to the corresponding Gutenstyle?

Reply:
The same is true for all our other guten-variables: https://github.com/Automattic/wp-calypso/blob/bb9101f7850fdd198abc24d1c45bcdf466d39cea/client/gutenberg/extensions/markdown/editor.scss#L1-L9

Reply:
I agree - let's go with the short-term solution for now - this is what the TODO is for - we'll address it in the future.

Comment added in 88a56f8df16814671e7864fa7a511ac6c38689be.

Reply:
Hmm, @simison has a point. What about adding another var for the opacity so they're all in one place at least?

Reply:
The reason why I didn't do it is because those were originally vars, pulled from Gutenberg's styles, while the opacity isn't. If that still sounds good to you, I can move it to a dedicated var.

Reply:
Nah, your rationale makes sense.